.gitignore

```diff
@@ -8,3 +8,4 @@
 /website/dist/
 /BenchFind.native
 /api-fileutils.docdir
+*.merlin
```

Comment:
I think it should be "/.merlin"

Can you also remove:
-/_build/
-/setup.data
-/setup.log
-/test.byte
-/BenchFind.native
-/api-fileutils.docdir

And ignore /*.install?

Reply:
There's merlin generated files in `src` and `test`, using `*.merlin` ignore all of them...

Reply:
I kept `_build` as it's generated by `dune`.